shell/gpu/gpu_surface_noop.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_GPU_GPU_SURFACE_NOOP_H_
+#define FLUTTER_SHELL_GPU_GPU_SURFACE_NOOP_H_
+
+#include <Metal/Metal.h>
+
+#include "flutter/flow/surface.h"
+#include "flutter/fml/macros.h"
+
+namespace flutter {
+
+class GPUSurfaceNoop : public Surface {
```

Comment:
A short docstring saying something like "Accepts rendering intent but doesn't render anything. Useful for tests that don't care about checking rendering output but still need to run a Flutter app." Or something like that.

Reply:
Done